clang/test/AST/HLSL/resource_binding_attr.hlsl

```diff
@@ -1,13 +1,15 @@
 // RUN: %clang_cc1 -triple dxil-pc-shadermodel6.3-library -x hlsl -ast-dump -o - %s | FileCheck %s
 
```

Comment:
Should we support
```
 float c : register(c10);
```
It is legal in dxc.

Reply:
yes